mdflynn/refactor-tractor-fitlit
# Feature/refactor scripts

## What is the change?
- This refactors multiple functions to adhere to SRP.
- Adds jQuery throughout scripts.js file and remove global querySelectors
- Adds updated sass dependencies

## What does it fix?
No fix this Pull Request.

## Is this a fix or a feature?
This is a feature.

## Where should the reviewer start?
Review the entire scripts.js file, there were updates to the entire file.

## How should this be tested?
Run ```npm start``` in your terminal. Then navigate to localhost:8080 to see the current site.
